microprofile/health-smallrye/src/main/resources/org/wildfly/extension/microprofile/health/LocalDescriptions.properties

```diff
@@ -0,0 +1,5 @@
+microprofile-health-smallrye=WildFly Extension for Eclipse MicroProfile Health With SmallRye
+microprofile-health-smallrye.add=Add the subsystem
+microprofile-health-smallrye.remove=Remove the subsystem
+microprofile-health-smallrye.check=Check the healthiness of the application server and its deployments
+microprofile-health-smallrye.security-enabled=True if authentication is required to access the HTTP endpoint.
```

Comment:
Perhaps add "on the management interface."?

If we later expose this on the undertow subsystem having this wording here gives us some freedom to use some different configuration for authentication there if needed. And if it turns out this attribute is applicable we can just remove these words.

No big deal because we could always add those words later if it turned out this attribute could only apply to the mgmt interface.

Reply:
fixed